pkg/epp/scheduling/types/types.go

```diff
@@ -43,11 +43,14 @@ func (r *LLMRequest) String() string {
 type Pod interface {
 	GetPod() *backendmetrics.Pod
 	GetMetrics() *backendmetrics.Metrics
-	SetScore(float64)
-	Score() float64
 	String() string
 }
 
+type ScoredPod struct {
+	Pod   Pod
```

Comment:
nit: I think embedding the Pod is better.

Reply:
good idea.
will update in a follow up PR not to delay this one.